### PR TITLE
Testing pytest failure output

### PR DESCRIPTION
**Pull request by trevor2718**
# Description

Cleaning up the pytest log output

Fixes [vanotify-team/624](https://app.zenhub.com/workspaces/va-notify-620d21369d810a00146ed9c8/issues/department-of-veterans-affairs/vanotify-team/624)

## Type of change

Please delete options that are not relevant.

- [x] Cleaning up the pytest log output

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by trevor2718** [comment]
See this for recent deploy (induced) error:
https://github.com/department-of-veterans-affairs/notification-api/actions/runs/3584681111/jobs/6031934116#step:3:42

File: lambda_functions/user_flows/user_flows_lambda.py [hunk 1]
```diff
@@ -7,6 +7,7 @@ def user_flows_handler(event, context):
     return pytest.main([
         "-p", "no:cacheprovider",
         "-s", "--verbose",
+        # "--tb=short",
```
[the remark this answers is not in the document]
Comment (k-macmillan) [reply lambda_functions/user_flows/user_flows_lambda.py:11]: No need to have these comments

**Review by k-macmillan — CHANGES_REQUESTED** [review]

File: lambda_functions/user_flows/conftest.py [hunk 2]
```diff
@@ -9,3 +9,21 @@ def pytest_generate_tests(metafunc):
...
+                args = entry.reprfuncargs.args
+                for idx, (name, value) in enumerate(args):
+                    if "service_test_api_key" in name:
+                        args[idx] = (name, "********")
+            if entry.reprlocals is not None:
+                lines = entry.reprlocals.lines
+                for idx, line in enumerate(lines):
+                    if line.startswith("service_test_api_key"):
+                        lines[idx] = "service_test_api_key          = '*********'"
+
```
Comment (cris-oddball) [inline lambda_functions/user_flows/conftest.py:29]: This works for `service_test_api_key`, but there is another variable in use by the tests, which if those test fail, could then print out that variable: `service_api_key`. So this code need to look for both variables.

**Review by cris-oddball — CHANGES_REQUESTED** [review]

**Review by cris-oddball — DISMISSED** [review]

**Review by EvanParish — APPROVED** [review]

File: lambda_functions/user_flows/test_retrieve_everything.py [hunk 3]
```diff
@@ -166,7 +166,8 @@ def test_send_email(notification_url, service_id, service_api_key, template_id):
 
     assert notification_status_response['status'] == desired_status
     assert notification_status_response['email_address'] is not None
-    assert notification_status_response['sent_by'] == 'govdelivery'
+    # below changed to "ses" per David
```
Comment (k-macmillan) [inline lambda_functions/user_flows/test_retrieve_everything.py:169]:
```suggestion
```

**Review by k-macmillan — CHANGES_REQUESTED** [review]

**Review by EvanParish — DISMISSED** [review]

**Review by cris-oddball — APPROVED** [review]

**Review by EvanParish — APPROVED** [review]

**Review by cris-oddball — APPROVED** [review]

**Pull request merged**